frontend/src/components/LandingPage/TeamCard/TeamCard.js

```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import {
+  Card, /*CardImg, CardText,*/ CardBody,
```

Comment:
What do you think about removing the commented out imports instead of just having them commented.

Reply:
I can do that, will push soon!